build.sbt

```diff
@@ -31,3 +34,4 @@ lazy val it = project
   .dependsOn(microservice % "test->test") // the "test->test" allows reusing test code and test dependencies
   .settings(DefaultBuildSettings.itSettings())
   .settings(libraryDependencies ++= AppDependencies.it)
+  .settings(Test / classLoaderLayeringStrategy := ClassLoaderLayeringStrategy.Flat)
```

Comment:
I think this setting may only be relevant if fork is false (https://www.scala-sbt.org/1.x/api/sbt/ClassLoaderLayeringStrategy.html).